src/framework/toolbox.js

```diff
@@ -16,6 +16,11 @@ var FrameworkToolbox = GObject.registerClass(class FrameworkToolbox extends Gtk.
         this._model = new RaModel();
 
         this._controlPanel.bindModel(this._model);
+
+        const provider = new Gtk.CssProvider();
+        provider.load_from_resource('/com/endlessm/HackToolbox/framework/application.css');
+        Gtk.StyleContext.add_provider_for_screen(Gdk.Screen.get_default(),
+            provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION + 1);
```

Comment:
Why is this added for the whole screen? It should only be tied to the lifecycle of the framework toolbox, so should either be added to its style context, or removed when the toolbox is destroyed.

Reply:
I was assuming that when the theming for the toolboxes is done, we will want to put more CSS rules in here that only apply to the framework toolbox.
I guess the best thing to do is to remove it when the window is destroyed.

Reply:
Yeah, that would probably work. The toolbox process is reused for different windows that are not just for the framework, that's why I wanted it to only apply to the framework toolbox.

Reply:
I was thinking that last night after I wrote this comment.
So my suggestion wouldn't work. We should add a `.framework` CSS class to the toplevel window and add the CSS rules to the main `application.css` instead.